lib/nfs_common.pm

```diff
@@ -271,8 +276,11 @@ sub check_y2_nfs_func {
         start_service($rw, $ro);
     }
     check_service($rw, $ro);
-    if ($stage eq 'before') {
-        stop_service();
+    stop_service();
+    if ($stage eq 'after') {
```

Comment:
Please add comments to describe why need these cmds.

Reply:
done